lib/sensors/src/MoistureSensor.cpp

```diff
@@ -35,8 +35,8 @@ uint8_t MoistureSensor::Read_HW_ID() {
 
   char check[1];
 
```

Comment:
undo all changes to this file by doing git checkout master lib/sensors/src/Moisture.cpp

Reply:
copy/pasted the contents of moisturesensor.cpp into my branch